public/data/deborshikun.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Deborshi Chakrabarti",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/deborshikun.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub!!!",
+      "url": "https://github.com/deborshikun",
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Learning about Open Source Contributions",
+      "date": "July 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started Open Source learning",
+      "url": "https://www.eddiejaoude.io/"
```

Comment:
Did you mean to link to Eddie’s site here? Or would you like to change it to one of your own?

Reply:
I don't have any websites to show to be honest so i thought let it be as it is. Should I change it to something else?

Reply:
You could remove it if you like @deborshikun, as you don't need to have a url . Thank you.

Reply:
Oh okay I got it!